### PR TITLE
dapps-staking pallet integration

### PR DESCRIPTION
**Pull request by shunsukew**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,11 @@
 
 Swanky node is the Substrate based blockchain configured to enable `pallet-contracts` and more features to help Wasm Smart Contract development.
 
+## Features
+- pallet-contracts enabled
```
Comment (PierreOssun) [inline README.md:6]: Can you specify which pallet-contract version will it targets? And if `unstable-feature` is enabled by default

File: README.md [hunk 2]
```diff
@@ -2,6 +2,11 @@
...
+- Instant seal and Manual seal is used for Consensus. No aura and grandpa.
```
Comment (PierreOssun) [inline README.md:7]:
```suggestion
- `grandpa` & `aura` consensus was removed. By default `instant-seal` is used - and `manual-seal` can also be used
```

**Comment by shunsukew** [comment]
@PierreOssun Thank you for the comment. I updated commented part of README file. This PR is for dapps-staking integration, I'll raise different PR to improve README later again!

File: runtime/src/lib.rs [hunk 3]
```diff
@@ -275,6 +287,69 @@ impl pallet_contracts::Config for Runtime {
 	type AddressGenerator = pallet_contracts::DefaultAddressGenerator;
 }
 
+parameter_types! {
+	pub const DappsStakingPalletId: PalletId = PalletId(*b"py/dpsst");
+	pub const BlockPerEra: BlockNumber = 4 * HOURS;
```
[the remark this answers is not in the document]
Comment (shunsukew) [reply runtime/src/lib.rs]: Okay!

File: runtime/src/lib.rs [hunk 4]
```diff
@@ -275,6 +287,69 @@ impl pallet_contracts::Config for Runtime {
...
+	pub const RegisterDeposit: Balance = 100 * UNIT;
+	pub const MaxNumberOfStakersPerContract: u32 = 2048;
+	pub const MinimumStakingAmount: Balance = 5 * UNIT;
+	pub const MinimumRemainingAmount: Balance = 1 * UNIT;
+	pub const MaxEraStakeValues: u32 = 5;
+	pub const MaxUnlockingChunks: u32 = 32;
```
[the remark this answers is not in the document]
Comment (shunsukew) [reply runtime/src/lib.rs:298]: Which one? MaxUnlockingChunks?

[the remark this answers is not in the document]
Comment (shunsukew) [reply runtime/src/lib.rs:298]: Seems yes MaxUnlockingChunks.
I copied values from Shibuya not Local somehow.
Please let me update to match values to Local runtime. I understand Shibuya also needs updates!

**Comment by shunsukew** [comment]
Updated All dapps-staking pallet parameters to match local runtime (not shibuya).

**Pull request merged**